frontend-react/src/AppRouter.tsx

```diff
@@ -41,6 +43,50 @@ export enum FeatureName {
 }
 
 export const AppRouter = () => {
+    const {
+        sessionStartTime,
+        setSessionStartTime,
+        sessionTimeAggregate,
+        setSessionTimeAggregate,
+    } = useSessionContext();
+
+    useEffect(() => {
+        const onUnload = () => {
+            trackAppInsightEvent("session", {
```

Comment:
Can we add [an enum](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/utils/Analytics.ts#L3) for this event name?

Reply:
Yes, please add to Analytics.ts -> EventName